src/index.html

```diff
@@ -16,5 +16,8 @@
   <header class="header">
     Header
   </header>
+  <footer class="footer">
```

Comment:
The footer appears directly below the header. Can you position it at the botto of the page?